index.js

```diff
@@ -146,17 +146,23 @@ KoaRoles.prototype.use2 = function (action, fn) {
 /**
  * @method KoaRoles#can
  * @param {String} action
+ * @example KoaRoles#can('actionAOnly'), KoaRoles#can('actionA', 'orActionBIsOK')
  */
-KoaRoles.prototype.can = function (action) {
+KoaRoles.prototype.can = function () {
+  var args = args2arr(arguments);
```

Comment:
@fool2fish Please use https://github.com/node-modules/utility/blob/master/lib/optimize.js#L64 instead of args2arr